src/cosalib/cmdlib.py

```diff
@@ -198,6 +198,14 @@ def import_ostree_commit(repo, commit, tarfile, force=False):
                                d, commit])
 
 
+def ostree_extract_efi(repo, commit, destdir):
+    """Given an OSTree commit, extract the EFI parts"""
+    ostreeefidir = "/usr/lib/ostree-boot/efi/EFI"
+    run_verbose(['/usr/bin/ostree', 'checkout', '--repo', repo,
+                 '--user-mode', '--subpath', ostreeefidir,
+                 commit, destdir])
+
+
```

Comment:
Would we ever want to use this function in other places? If not should we just put it inside `cmd-buildextend-installer`

Reply:
As the commit says, this is prep for using this elsewhere.  But yeah, we can keep it here for now then move it later.

Reply:
ahh - didn't realize that definitely can leave it in the lib then. sorry for the noise